### ISSUE TITLE
White blank page in preview qgis2web

### ISSUE THREAD

**Issue by Hosaana** [qgis2web/qgis2web#905]
Hello,
I have a white blank page in preview qgis2web and when I am trying to export I receive a white empty html as well.Can you please help me to fix that problem.On attachment,a screenshoot of my  of QGIS,and the generated folder.

[Thanks]

![image](https://user-images.githubusercontent.com/50334077/87219748-05b5cf80-c32c-11ea-8f86-2c048a96d43f.png)

![Screen Shot 2020-07-11 at 4 08 18 AM](https://user-images.githubusercontent.com/50334077/87219768-3138ba00-c32c-11ea-96d0-643af341d639.png)

**Comment by Hosaana** [comment]
Also this is the blank white page in preview

![Screen Shot 2020-07-11 at 4 16 20 AM](https://user-images.githubusercontent.com/50334077/87219988-23843400-c32e-11ea-9359-9382283cb6ed.png)

**Comment by tomchadwin** [comment]
Please upload your project and layers files here. Without them, it's impossible to know what the specific problem is. Simplify down to a single layer if you can.

**Comment by Hosaana** [comment]
Hello Tom,

I tried with Leaflet instead of openlayers and it worked,below is the
screenshot of my qgis2web preview

[image: image.png]


 However,I have some issues that I will need your help :

1)How can I add the checkbox on the sublayers as in the image below

[image: image.png]

2)Is there a way to add layers in the webmap in the future,for example  a
button  to add new datas or download a table of datas and add it to the map?
3)If number 2 is not possible,is there a way to do it manually and  change
entry data?
4)How to add a map title and show different sublayers only when I check a
layer which contain them like on the map here:
http://gis.gscplanning.com/snow-removal/===>snow removal layers

5)Is there a way to add background maps like in the map above in the link?

In short,I need to make a map which have all functionalities like in  that
link:http://gis.gscplanning.com/snow-removal/

Looking forward to hearing from you,

With regards,

Hosiana

**Comment by Hosaana** [comment]
Hello, Can you please help me with these issues: 1)I can only see my map
layers when I click on zoom in ,the same in my html map?how to fix it so
that the preview and html can open directly the map ? 2)Is there a way to
add entry data in the html map?Let say the user wants to visualize other
data?

Regards,

Hosiana

**Comment by Hosaana** [comment]
Soory I forgot to show you the page in preview qgis2web:

![image](https://user-images.githubusercontent.com/50334077/87893041-a717e200-ca0c-11ea-95c8-0a21ce623495.png)

**Comment by tomchadwin** [comment]
Please upload your project and layers files here. Without them, it's impossible to know what the specific problem is. Simplify down to a single layer if you can.
